Makefile

```diff
@@ -47,8 +57,13 @@ shumway: build/shumway.js
 build/shumway.js: $(SHUMWAY_SRCS)
 	node tools/tsc.js --sourcemap --target ES5 shumway/references.ts --out build/shumway.js
 
+CONFIG_BUILD_CONTENT := "config.release = ${RELEASE};"
+OLD_CONFIG_BUILD_CONTENT := "$(if $(wildcard config/build.js),$(shell cat config/build.js),)"
+
 config-build:
+ifneq ($(CONFIG_BUILD_CONTENT),$(OLD_CONFIG_BUILD_CONTENT))
 	echo "config.release = ${RELEASE};" > config/build.js
+endif
```

Comment:
Hmm, if we're also going to modify the way that the *config-build* target produces config/build.js, we might as well start change it to use a preprocessed file (config/build.js.in).